### DISCUSSION TITLE
Solve the error reporting problem of libpqxx under nvim

### DISCUSSION THREAD

**Discussion by RedTrait** [jtv/libpqxx#830] [General]
history
https://github.com/jtv/libpqxx/issues/829

**Comment by RedTrait** [comment]
I changed all the files and search suffix names to hpp, and checked each file one by one. Finally, I found that I was missing a header file.`libpq-forward.hpp`,  Why am I missing this file
![image](https://github.com/jtv/libpqxx/assets/45113206/ac2cf08b-9088-424e-bb52-f8448aa65af3)

**Reply by tt4g** [reply]
Since `libpq-forward.hxx` exists in this repository: https://github.com/jtv/libpqxx/blob/7.9.0/include/pqxx/internal/libpq-forward.hxx
the missing file is probably a problem with your machine.

**Comment by jtv** [comment]
It sounds to me as if either:
1. this file went missing or was skipped in your renaming; or
2. nvim is somehow looking at the include directory in the build directory, but not at the one in the source tree.  It needs both.
